server/src/metadata.rs

```diff
@@ -365,4 +365,19 @@ pub mod entities_metadata_query {
         .map(|id| LinkedNode { id })
         .collect()
     }
+
+    impl Serialize for Organization {
+        fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+        where
+            S: Serializer,
+        {
+            let mut map = serializer.serialize_map(Some(3))?;
+
+            map.serialize_entry("id", "https://serlo.org/#organization")?;
+            map.serialize_entry("type", "Organization")?;
+            map.serialize_entry("name", "Serlo Education e.V.")?;
+
+            map.end()
+        }
+    }
```

Comment:
As said in the meeting: I am flexible on another solution. An alternative I would do

```rust
fn query() {
  ...
  maintainer: create_serlo_organization_json(),
  publisher: vec![create_serlo_organization_json()],
}

fn create_serlo_organization() {
  return json!({

  })
}
```

@AndreasHuber @hugotiburtino What are your opinions?

Reply:
I think also that the `serialize` is a overkill, it would make more sense if there were more logic. 
So, I'm in favor of doing `create_serlo_organization`, but if it by chance throws a lot of compiler error, taking longer than one hour to implement it, we could use the serialize option, since it is already done.

Reply:
Done. Thanks to #421 I could also test whether this has any changes on the benachmark: Nope -> So I am going to merge it with the change